src/adapter/src/catalog/apply.rs

```diff
@@ -179,23 +250,15 @@ impl CatalogState {
     #[instrument]
     pub(crate) async fn apply_updates_for_bootstrap(&mut self, updates: Vec<StateUpdate>) {
         let SortedUpdates {
-            pre_builtin_view_updates,
-            builtin_view_updates,
-            post_builtin_view_updates,
+            pre_builtin_view_additions,
+            builtin_view_additions,
+            post_builtin_view_additions,
         } = sort_updates(updates);
-        for StateUpdate { kind, diff } in pre_builtin_view_updates {
-            assert_eq!(
-                diff, 1,
-                "initial catalog updates should be consolidated: ({kind:?}, {diff:?})"
-            );
+        for StateUpdate { kind, diff } in pre_builtin_view_additions {
```

Comment:
At first I was a bit confused by the `pre_builtin_view_additions` naming here because AFAICT it also includes retractions? Should this be renamed to `pre_builtin_view_updates`?

Reply:
I would love a better structure or naming convention for this if you have any ideas because it's really confusing. It includes all updates that are before the "builtin view additions" so it's pre-"builtin view additions". Those updates do include retractions and additions (including builtin view retractions).

Reply:
Actually I thought of a simplification that I'll push in a followup PR.